subprojects/gradle/instrumentation-tests-test-fixtures/src/main/kotlin/com/avito/instrumentation/reservation/client/kubernetes/FakeDeploymentNameGenerator.kt

```diff
@@ -0,0 +1,9 @@
+package com.avito.instrumentation.reservation.client.kubernetes
+
+class FakeDeploymentNameGenerator(
+    private val variablePart: String = "integration-test"
```

Comment:
I don't understand the name `variablePart`

Reply:
let's rename it then;
how about `deploymentName`, `name`, `customName`?

Reply:
`deploymentNamePrefix`?

Reply:
`deploymentName` already in class name, `prefix` is in fact `postfix`